code/modules/atmospherics/machinery/components/components_base.dm

```diff
@@ -20,27 +20,16 @@
 		A.volume = 200
 		airs[i] = A
 
-/obj/machinery/atmospherics/components/Initialize()
-	. = ..()
-
-	RegisterSignal(src, COMSIG_OBJ_HIDE, .proc/hide_pipe)
```

Comment:
Components worked kinda weirdly. The pipe would be hidden but the component not, thus having sprites weirdly clipping through the floor. I just turned it off because it really isn't worth the hassle and it looks absolutely terrible.

Reply:
errr, nvm this includes vents fuck